projects/plugins/super-cache/tests/e2e/jest.config.js

```diff
@@ -1,10 +1,8 @@
-import path from 'path';
 import { fileURLToPath } from 'url';
-import dotenv from 'dotenv';
+import * as dotenv from 'dotenv';
```

Comment:
Why that and not this?
```suggestion
import { config } from 'dotenv';
```

Reply:
Same as the emailValidator one.